src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs

```diff
@@ -9,6 +9,353 @@ namespace System.Text.Json
 {
     public sealed partial class Utf8JsonWriter
     {
+        /// <summary>
+        /// Writes the pre-encoded property name (as a JSON string) as the first part of a name/value pair of a JSON object.
+        /// <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        /// </summary>
+        /// <remarks>
+        /// The property name should already be escaped when the instance of <see cref="JsonEncodedText"/> was created.
```

Comment:
"The property name should already be escaped when the instance of JsonEncodedText was created." is strange grammatically.  What is it trying to convey?  Is there something the user needed to do but didn't?  Is this stating a fact about something that must be the case?

Reply:
I agree this remark is weird.  It's like taking an `int` and remarking the value has to be between `int.MinValue` and `int.MaxValue` inclusive... it's a truth provided by the type system.